src/util/createResponsiveStoreEnhancer.js

```diff
@@ -3,15 +3,34 @@ import addEventHandlers from './addEventHandlers'
 
 /**
  * Creates a store enhancer based off an (optional) throttle time.
- * @arg {number} [throttleTime=100] - Throttle time (in miliseconds) for the
- * window resize event handler.
+ * @arg {(object|number)} [options={throttleTime,calculateStateInitially}]
+ * - Options object or the throttle time.
+ * @arg {number} [options.throttleTime=100] - Throttle time (in miliseconds) for
+ * the window resize event handler.
+ * @arg {boolean} [options.calculateStateInitially=true] - True if the responsive
+ * state must be calculated initially, false otherwise.
  * @returns {function} - The store enhancer (which adds event listeners to
  * dispatch actions on window resize).
  */
-export default (throttleTime = 100) =>
+export default (options = {throttleTime: 100, calculateStateInitially: true}) => {
+    // options object normalization
+    const normalizedOptions = {
+        throttleTime: 100,
+        calculateStateInitially: true,
+    }
+    if (typeof options === 'number') {
```

Comment:
Thanks for going through the trouble of maintaining backwards compatibility 👍 
 
However, I am a bit wary of the potentially confusing api since the function accepts two different kinds of types. Another option that avoids the typechecking is to pass each config option as individual arguments. It's definitely not ideal when the number of arguments grows, but 2 isn't unwieldy. Also, we should probably just bump the version number and move to just supporting objects when the time comes (I probably should have done that in the first place).